.github/workflows/lint-rules.yml

```diff
@@ -0,0 +1,30 @@
+name: Update lint_rules list
+
+on: push
```

Comment:
Need to change to trigger on "push to main" only, when oxc_linter has a change.

Reply:
👍🏻 

👇🏻  was my original intention(currently commented out for debug)

```yml
on:
  pull_request:
    branches:
      - main
    types:
      - closed

jobs:
  lint_rules:
    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'A-linter')
```

or you prefer like this?

```yml
  push:
    branches:
      - main
    paths:
      - 'crates/oxc_linter/**'
```

Reply:
(This one looks simpler. 🤔 This one looks simpler. I'll take this one.)